#gain: only roll, pitch, yaw_d (throttle and yaw pi is calculated in PC side)

# roll
uint16 roll_p
uint16 roll_i
uint16 roll_d

# pitch
uint16 pitch_p
uint16 pitch_i
uint16 pitch_d

# yaw
uint16 yaw_d
